### ISSUE TITLE
Support Raspbian Stretch

### ISSUE THREAD

**Issue by luckygulli** [piksel/phantomjs-raspberrypi#4]
After installing, i got this error :
```
$ phantomjs examples/hello.js
phantomjs: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory
```
I had to :
```
sudo apt-get install libssl1.0.0
wget http://launchpadlibrarian.net/201335791/libicu52_52.1-8_armhf.deb
sudo dpkg -i libicu52_52.1-8_armhf.deb
rm -rf libicu52_52.1-8_armhf.deb
```

I would suggest to add it in the readme

**Comment by piksel** [comment]
Did you try to install `libssl1.0.0` via `apt`?

```
sudo apt install libssl1.0.0
```
Why the libicu52 package?

**Comment by luckygulli** [comment]
Yes, I had to follow this tutorial to add a new repository to get `libssl1.0.0` : https://www.stewright.me/2016/03/turn-raspberry-pi-3-php-7-powered-web-server/ because raspbian told me he this library was found but not available for download.

Then, I got the error
```$ phantomjs examples/hello.js
phantomjs: error while loading shared libraries: libicui18n.so.52: cannot open shared object file: No such file or directory```
But this time, the package libicu52 was even not found in the repository (The same issue with Ubuntu https://github.com/Pyppe/phantomjs2.0-ubuntu14.04x64/issues/1)

So I did what I mentioned in my first post. (I will edit to add that I run `sudo apt-get install libssl1.0.0` first.)

**Comment by piksel** [comment]
I didn't need to install any of those on a fresh install of raspian on a pi0 and on a pi3. I might've installed some other packages this, so I will investigate. But definitely no custom apt sources tho.

_Sent from my Huawei ALE-L21 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by luckygulli** [comment]
To help you to investigate, the first time I've installed it (one week ago), I didn't need to do so many things (I added the custom apt-source, but for another tool, and I didn't have to download any package manually).
When I reinstalled it 2 days ago, I needed to do this.

The biggest difference between the two times was my OS:
- The first time, it was [Raspbian Jessie Lite](https://raspbian-france.fr/telechargements/) ;
- The second time, it was [Raspbian Stretch Lite](https://www.raspberrypi.org/downloads/raspbian/).

**Comment by piksel** [comment]
I have confirmed the problem in Raspbian Stretch.

It uses libicu57 (binaries require v52) and libssl1.1 or 1.0.2 (binaries require v1.0.0).

I will have to do a new compile for stretch, but unfortunately I don't have any SSDs to spare right now so it might take a while!

**Comment by piksel** [comment]
Fixed by 65d1e2a 
Download new binary release here:
https://github.com/piksel/phantomjs-raspberrypi/releases/tag/v2.1.1-switch
